tst/testinstall/opers/NormalSubgroups.tst

```diff
@@ -0,0 +1,113 @@
+gap> START_TEST("NormalSubgroups.tst");
+
+# Natural symmetric groups
+gap> NormalSubgroups(SymmetricGroup(0)) = [Group(())];
+true
+gap> NormalSubgroups(SymmetricGroup(1)) = [Group(())];
+true
+gap> Set(NormalSubgroups(SymmetricGroup(2))) =
+> Set([Group(()), SymmetricGroup(2)]);
+true
+gap> Set(NormalSubgroups(SymmetricGroup(3))) =
+> Set([Group(()), AlternatingGroup(3), SymmetricGroup(3)]);
+true
+gap> Set(NormalSubgroups(SymmetricGroup(4))) =
+> Set([Group(()), Group((1,2)(3,4), (1,3)(2,4)),
+>  AlternatingGroup(4), SymmetricGroup(4)]);
+true
+gap> Set(NormalSubgroups(SymmetricGroup(5))) =
+> Set([Group(()), AlternatingGroup(5), SymmetricGroup(5)]);
+true
+gap> Set(NormalSubgroups(SymmetricGroup(6))) =
+> Set([Group(()), AlternatingGroup(6), SymmetricGroup(6)]);
+true
+gap> Set(NormalSubgroups(SymmetricGroup(100))) =
+> Set([Group(()), AlternatingGroup(100), SymmetricGroup(100)]);
+true
+gap> Set(NormalSubgroups(SymmetricGroup([3,7,4,11,70]))) =
+> Set([Group(()), AlternatingGroup([3, 4, 7, 11, 70]),
+>      SymmetricGroup([3, 4, 7, 11, 70])]);
+true
+gap> Set(NormalSubgroups(SymmetricGroup([7,4,13,21]))) =
+> Set([Group(()), Group((4,7)(13,21), (4,13)(7,21)),
+>      AlternatingGroup([4, 7, 13, 21]), SymmetricGroup([4, 7, 13, 21])]);
+true
+gap> NormalSubgroups(SymmetricGroup([42])) = [Group(())];
+true
+gap> NormalSubgroups(SymmetricGroup([])) = [Group(())];
+true
+gap> G := Group((7,3,5,11), (11,3), (5,3));;
+gap> IsNaturalSymmetricGroup(G);
+true
+gap> Set(NormalSubgroups(G)) =
+> Set([Group(()), Group((3,5)(7,11), (3,7)(5,11)),
+>      AlternatingGroup([3, 5, 7, 11]), SymmetricGroup([3, 5, 7, 11])]);
+true
+
+# Non-natural symmetric groups
+gap> S4 := Group((1,2,3,4), (1,2));;
+gap> hom := ActionHomomorphism(g, Arrangements([1..4], 4), OnTuples);;
```

Comment:
change `g` to `S4` here (else the test fails when run on its own).

Reply:
Thank you!  It's always the little things...